misc/config_tools/scenario_config/scenario_cfg_gen.py

```diff
@@ -6,6 +6,8 @@
 import os
 import sys
 import copy
+import xmlschema
```

Comment:
The idea is to wrap `import xmlschema` in a try-except block so that we can skip the schema checking if the Python library is not installed. Kindly note that this script is imported in the build process where xmlschema may not be available.

Reply:
@yuchuyang please refer junjie's comments, it has led to build fail now, thanks